packages/squiggle-lang/src/public/SqPlot.ts

```diff
@@ -61,4 +68,37 @@ export class SqFnPlot extends SqAbstractPlot<"fn"> {
   }
 }
 
-export type SqPlot = SqDistributionsPlot | SqFnPlot;
+export class SqJointPlot extends SqAbstractPlot<"joint"> {
+  tag = "joint" as const;
+
+  get yDist() {
+    return wrapDistribution(this._value.xDist);
+  }
+  get xDist() {
+    return wrapDistribution(this._value.yDist);
+  }
```

Comment:
Why do you need these two methods? For future functionality?

Reply:
Initially I put the code for `points()` in the components, before it became clear that it belongs to `squiggle-lang`. But since I already wrote those methods, I decided that they might be useful later.